core_project_message.md

```diff
@@ -0,0 +1,41 @@
+Operate software in a production grade environment
+What is Operate First?
+
+Operate First is a concept where open source developers bring their projects to a production cloud during development where they can interact with a cloud provider’s operators and gain valuable feedback on the operational considerations of their project.
+The motivation for the Operate First initiative is the shift of focus from software availability to operating services.
+Open source removed the access to software as a limiting factor for businesses and individuals.
+However, with software proliferating into every aspect of the business—and our world in general—resulting in ever growing complexity of software stacks, the challenge is now operationalizing software.
+
+Why Operate First?
+
+  "The ability to have hands on the MOC environment while installing two open source project releases was of real value.
+Our teams were able to resolve critical interdependencies that the test suites had not found.
+I could not imagine trying to work with a customer to debug this problem."
+    - An operate first member
+
+“Operate First” is inspired by Red Hat’s “Upstream First” philosophy, which emphasizes working in the open, even for Red Hat products.
+Operate First tries to extend “working in the open” to a part of the codebase that teams typically keep closed, even when deploying open source projects: their operations code.
+
+Key Personas
+- Developer/QE
+- Clouder Operator/SRE & SysAdmin
```

Comment:
```suggestion
- Cloud Operator/SRE & SysAdmin
```

Just so we're not confused with [clowder](https://github.com/RedHatInsights/clowder) project